client/tests/acceptance/pas-form-validation-messages-display-test.js

```diff
@@ -28,8 +28,6 @@ module('Acceptance | pas form validation messages display', function(hooks) {
     await visit('/packages/1/edit');
 
     assert.dom('[data-test-submit-button]').hasAttribute('disabled');
-    assert.dom('[data-test-save-button]').hasNoAttribute('disabled');
```

Comment:
@trbmcginnis this test started failing - I think because this ^ was in here to accomodate the beginning dirty state that was caused by having the `addApplicants()` trigggered when applicants was empty.  

If that's not the case, then I'll dig into why this assertion started failing.

Reply:
Yep I incorporated that "dirty-when-land-on-page-so-need-to-press-save-button" situation to my test! So feel free to edit based on your changes

Reply:
Sweet - glad this PR lets us remove this weird unintended behavior from our tests :-)